__tests__/app.test.js

```diff
@@ -122,3 +122,47 @@ describe('/api/articles/:article_id/comments', () => {
   });
 });
 
+describe('POST /api/articles/:article_id/comments', () => {
+  test('POST:201 adds a comment to the comments table where the user exists', () => {
+    const newComment = {
+      body: "Test Comment",
+      author: "icellusedkars",
+    };
+    return request(app)
+      .post('/api/articles/3/comments').send(newComment)
+      .expect(201)
+      .then((response) => {
+        expect(response.body.result.body).toBe("Test Comment");
+        expect(response.body.result.author).toBe("icellusedkars");
+        expect(response.body.result.article_id).toBe(3);
+      });
+  });
+  test('POST:404 adds a comment to the comments table when the user does not exist', () => {
```

Comment:
This test is good but the description is a little confusing as it does NOT add a comment when the user doesn't exist